handlers/geolocation/geolocation.go

```diff
@@ -128,6 +128,15 @@ func (c *GeolocationHandlersCollection) RedirectHandler() httprouter.Handle {
 	}
 }
 
+// RedirectConstPathHandler redirects const path into the self catalyst node
+func (c *GeolocationHandlersCollection) RedirectConstPathHandler() httprouter.Handle {
```

Comment:
Is this redirecting to itself? I'm not clear on what redirect issue this is fixing but could it just not redirect and handle the request?

Reply:
Ahh, yeah that's confusing. I've updated the description. PTAL.

Technically, yes, we don't need any redirect, but then each analytics log for the same viewing session may be sent to a different catalyst node. We don't want this because of the performance (some playback-related data is cached in the catalyst node).

Reply:
Gotcha, makes sense now thanks!

Reply:
@leszko just might be worth to document it here, in the code.

Reply:
Updated in 4bbf94c9b6036dd5edfa8427d7d36d677c6a759f